src/lib.rs

```diff
@@ -128,6 +182,73 @@ fn transaction_client_begin_pessimistic(client: &TransactionClient) -> Result<Bo
     }))
 }
 
+fn raw_get(cli: &RawKVClient, key: &CxxString, timeout_ms: u32) -> Result<OptionalValue> {
```

Comment:
`timeout_ms` not used?

Reply:
Yes, and I record here https://github.com/tikv/client-cpp/pull/17/files#diff-b1a35a68f14e696205874893c07fd24fdb88882b47c23cc0e0c80a30c7d53759R198 , I will try it implement it in another PR